.github/workflows/prepare_review_stage.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          submodules: true
```

Comment:
The bot doesn't need access to the submodules

Reply:
Fixed : f406515